gatsby-node.js

```diff
@@ -156,12 +156,13 @@ exports.createPages = async ({ graphql, actions }) => {
       query {
         allPagesJson(
           filter: { investigation: { eq: "solar-system" } }
-          sort: { fields: [order, investigation], order: ASC }
+          sort: { fields: [sectionId, order, investigation], order: ASC }
```

Comment:
Added sectionId as the primary sort field, will fall back to order. This maintains backward compatibility (un-sectioned investigations will be sorted by order) and simplifies order so pages are not placed in relation to the entire pageset, but just to their own section.